src/libcollections/string.rs

```diff
@@ -1301,6 +1301,63 @@ impl String {
         }
     }
 
+    /// Create a splicing iterator that removes the specified range in the string,
+    /// replaces by the giving string, and yields the removed chars.
+    /// The given string doesn’t need to be the same length as the range.
+    ///
+    /// Note: The element range is removed even if the iterator is not
+    /// consumed until the end.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the starting point or end point do not lie on a [`char`]
+    /// boundary, or if they're out of bounds.
+    ///
+    /// [`char`]: ../../std/primitive.char.html
+    ///
+    /// # Examples
+    ///
+    /// Basic usage:
+    ///
+    /// ```
+    /// # #![feature(splice)]
```

Comment:
For unstable documentation we tend to not hide `#![feature]` directives as they're required for it to build anyway